### PR TITLE
Booped

### PR DESCRIPTION
**Pull request by jamiemlawrence**
Added initial images to photo gallery page and created shared nav button.
Screenshot 2024-09-24 at 10 57 15 PM

### CODE REVIEW

File: src/PhotoPage.tsx [hunk 1]
```diff
@@ -1,22 +1,111 @@
...
+  {
+    img: './gallery/6.jpg',
+    alt: 'image1'
+  },
+  {
+    img: './gallery/7.jpg',
+    alt: 'image1'
+  },
+]
+
```
Comment (AnselSmolund) [inline src/PhotoPage.tsx]: Hey James, lets move this array of static data into a different file. Maybe called photoData.ts or something?

File: src/PhotoPage.tsx [hunk 2]
```diff
@@ -1,22 +1,111 @@
...
+  const isMediumScreen = useMediaQuery('(max-width:900px)');
+  const isLargeScreen = useMediaQuery('(max-width:1200px)');
+  const getCols = () => {
+    if(isSmallScreen){
+      return 1;
+    } else if(isMediumScreen){
+      return 2;
+    } else if(isLargeScreen){
+      return 3;
+    } else{
```
Comment (AnselSmolund) [inline src/PhotoPage.tsx]: nit: put a space in between else and { 
```suggestion
    } else {
```

File: src/PhotoPage.tsx [hunk 3]
```diff
@@ -1,22 +1,111 @@
...
+            alt={item.alt}
+          />
+        </ImageListItem>
+      ))}
+      </ImageList>
+      <Typography sx={{ paddingTop: 4 }} variant="h3" color={MAIN_COLOR}>
+        STILL UPLOADING... COME BACK SOON FOR MORE
       </Typography>
+      
+
```
Comment (AnselSmolund) [inline src/PhotoPage.tsx]: REMVOE THESE EMPTY LINES. IDIOT!

**Review by AnselSmolund — APPROVED** [review]
Overall, LGTM 🔥

**Comment by AnselSmolund** [comment]
can you please add screenshots of your UI changes to the photo page james?

**Comment by AnselSmolund** [comment]
can i see what this would look like on mobile?

**Comment by jamiemlawrence** [comment]
> can i see what this would look like on mobile?

Sure thing, here is what it looks like on the brand spanking new apple iphone 14 pro max
Screenshot 2024-09-24 at 10 58 30 PM

**Pull request merged**